.github/workflows/e2e.yaml

```diff
@@ -80,6 +80,28 @@ jobs:
           name: logs
           path: logs
 
+  FeatureGroup03:
```

Comment:
@kezhenxu94 I have a doubt about the group naming principle. What are they? Why all these say `FeatureGroupxx`, and the last one shows `Gateway`?

Reply:
> @kezhenxu94 I have a doubt about the group naming principle. What are they? Why all these say `FeatureGroupxx`, and the last one shows `Gateway`?

The `Gateway` should be also renamed to `FeatureGroupXX` I think, if @wu-sheng you have better suggestion, I'm good to follow :) as for the `PHP` job, as an individual language agent, I prefer to set up another workflow, named `PHP Agent` or `PHP SDK`, WDYT

Reply:
I prefer it keeps in a separated group, to avoid unexpected CI process conflict with other tests. Also, most PMC know little about the PHP agent, if conflict occurs, it is hard to follow.

@heyanlong Please create `e2e.php.yaml`.
@kezhenxu94 , please rename to `FeatureGroupXX`, at least keep consistent. I don't have better idea for now.